Quadriga/src/main/webapp/WEB-INF/views/status_transform.jsp

```diff
@@ -0,0 +1,6 @@
+<META HTTP-EQUIV="refresh" CONTENT="5;URL=${pageContext.servletContext.contextPath}/auth/transformation/result/status">
+
+<h1> Your transformation is being generated... </h1>
+
+<p>The resulting graph of the transformation is being downloaded.
```

Comment:
That's not the page for downloading, is it? So it shouldn't say that the graph is being downloaded.